konfera/tests/test_models.py

```diff
@@ -133,6 +133,28 @@ def test_string_representation(self):
             '%s %s %s' % (dict(TITLE_CHOICES)[entry.title], entry.first_name, entry.last_name)
         )
 
+    def test_automatic_order_generator(self):
+        location = models.Location(title='test_title', street='test_street', city='test_city', postcode=000000,
+                                   state='test_state', capacity=20)
+        location.save()
+        event = models.Event(title='test_event', description='test', event_type='meetup', status='published',
```

Comment:
Please consider importing status and use const. In test I dont mind that much, but it is good practice to use const as you are preventing typos, and if in future we need to change it, we'll do it at one place.